### DISCUSSION TITLE
Will g helper work on MSI computer?

### DISCUSSION THREAD

**Discussion by attakuart** [seerge/g-helper#1842] [Q&A]
I have an MSI computer, will Ghelper work?

**Comment by seerge** [comment]
@attakuart it will launch but won't offer you much, as it mostly relies to asus proprietary stuff :)

You will be able to swap between windows power modes and overclock your NVidia GPU probably

**Reply by chorong761** [reply]
Would there be any chance of a MSI version of GHelper being developed?

**Reply by seerge** [reply]
Well, i don't have MSI device :) and have no idea how all this proprietary things are done there

**Reply by sonofcopper** [reply]
@seerge Hello Seerge, we need to change cpu watt like asus profiles'. Because when i using in  extreme mod i am closing MMIO (Dynamic PL1) because MSI has 55 watt limit. But if i do this, when i change the balance mod. My fans' slowly and my computer is very hot. Help us please !
